wiki/Beatmap/Pattern/Mania/Long Notes/en.md

```diff
@@ -0,0 +1,29 @@
+---
+stub: true
+tags:
+  - jacks
+---
+
+# Long Notes
+
+Also referred to in-game as holds, **long notes** are notes that require you to hold and release them afterwards. They are often referred to as LNs for short and are used to represent prolonged sounds in the music.
+
+![](img/ln.png)
+
+**Releases** are known to be their own term. It refers to the ending of a **long note**, the part that requires you to release the key.
+
+## Shields
+
+**Shields** occur when a note is placed consecutively before a long note in the same column. They play very similarly to [minijacks](/wiki/Beatmap/Pattern/Mania/Jacks/en.md#Minijack).
+
+![](img/shield.png)
+
+**Reverse shields** occur when a consecutive note follows a long note's release.
```

Comment:
One of them clarify that they are in the same column but the other one does not. I would say mentioning the same column for both is better but being consistent should be enough.

Reply:
```suggestion
**Shields** occur when a note is placed consecutively before a long note in the same column. They play very similarly to [minijacks](/wiki/Beatmap/Pattern/Mania/Jacks/en.md#Minijack).

![](img/shield.png)

**Reverse shields** occur when a consecutive note follows a long note's release within the same column.
```